frontend/src/container/NewDashboard/DashboardVariablesSelection/VariableItem.tsx

```diff
@@ -69,6 +74,10 @@ function VariableItem({
 		getOptions();
 	}, [getOptions]);
 
+	const onRefreshHandler = (): void => {
+		updateTimeInterval(getDefaultOption(location.pathname));
```

Comment:
do not access the global if you want pathname grab from useLocation hook

Reply:
Updated My PR With this suggested changes @palashgdev